docs/operational_guide/placement_configuration.md

```diff
@@ -136,6 +136,8 @@ After sending the delete command you will need to wait for the M3DB cluster to r
 
 #### Replacing a Node
```

Comment:
Can we add similar sections for add and remove? Basically in the add section say its not recommended to make new nodes seed nodes and if you find yourself in a situation where you need more than 3 M3DB seed nodes you should just run an external cluster and then something similar for remove?